README.md

```diff
@@ -38,7 +38,9 @@ render(<App />, document.getElementById('root'));
 ### [ModalComponent, openFunc, closeFunc, isOpenBool] = useModal(domNode?, { preventScroll?, focusTrapOptions?, showCloseButton?, renderCloseButton? })
 
 `ModalComponent`
+Type: React.FC<{ title?: React.ReactNode; description?: React.ReactNode, children?: React.ReactNode }>
```

Comment:
Now that title and description can be passed, type definitions have been added to README as well.